src/TLuaInterpreter.cpp

```diff
@@ -2361,6 +2361,25 @@ int TLuaInterpreter::disableKey( lua_State *L )
     return 1;
 }
 
+int TLuaInterpreter::killKey( lua_State *L )
+{
+    string luaSendText="";
+    if( ! lua_isstring( L, 1 ) )
+    {
+        lua_pushstring( L, "killKey: wrong argument type" );
```

Comment:
While looking into this, I also found on line 3201 the following variant. (I typed it in, as vmware stopped sharing my clipboard. this might not be exactly the line, but more or less)

`lua_pushfstring( L, tr( "startLogging: bad argument #1 (turn logging on/off, as boolean expected, got %s)", luaL_typename(L,1) ).toUtf8().constData() );`

This syntax seems more clear to me, but I assume there's a good reason to use the .arg() call instead? If so, I can fix that line while I'm at it.